New css, and the ability to change between servers

Bertogim opened the pull request:
Added:
- The ability to switch between servers
- More accurate CSS
- User chat profile pictures (PFP)

The only thing I’m unsure how to add is server icons because I need to edit more than just the JS and HTML...

![image](https://github.com/user-attachments/assets/fb953f33-2cb8-4e1f-bbe8-fd489c0e0e22)
![image](https://github.com/user-attachments/assets/6ebe9c76-5a70-4fa9-b950-5f51bd1eff5a)

chylex commented:
Thanks, I'll look through it soon, but I have a few notes right now:

- The viewer cannot have a dependency on a third-party CDN, it needs to work completely offline and have a reasonable set of fallback fonts. The viewer can reference the gg sans font if the user installs it into their operating system, but the font is proprietary so I cannot bundle it, and I don't want to depend on `cdnfonts`.
- I don't think including the Discord logo in the viewer is an allowed use of their branding.
- I don't think links should open in new tabs by default, that way users always have the option to choose whether to left-click to open in the same tab, or middle-click to open in a new tab. Making all links open in new tabs removes the first option.
- I can look into adding support for saving server icons before I merge that part of the PR.

Bertogim commented:
- True, i made fonts into local files
- Fixed (Asked chatgpt for a svg)
- Reverted, sorry i did not know what i was thinking
- That would be awesome!

I also removed some console.log i left

chylex commented:
> True, i made fonts into local files

Unfortunately I cannot include the files either, Discord explicitly does not allow anyone else to use the fonts.

> Fixed (Asked chatgpt for a svg)

ChatGPT cannot tell what the license of the icon it generated (or possibly stole) is, I will come up with a replacement or draw my own one if I need to.

Bertogim commented:
- I thought the font was just public, removed gg-sans
- So true, we cant know if its stole, replaced it with DM / Direct Messages (You can replace this as you wish)

chylex commented:
I added code to store server icons, when I have time I'll integrate it into the PR and clean it up.

Bertogim committed:
Undo??

Bertogim committed:
Merge branch 'master' of https://github.com/chylex/Discord-History-Tracker

Bertogim committed:
Delete .vs/Discord-History-Tracker/v17/DocumentLayout.json

Bertogim committed:
Update

Bertogim committed:
Updatee

Bertogim committed:
Merge branch 'master' of https://github.com/Bertogim/Discord-History-Tracker

Bertogim committed:
Merge branch 'chylex:master' into master

Bertogim committed:
Update

Bertogim committed:
Merge branch 'master' of https://github.com/Bertogim/Discord-History-Tracker

Bertogim committed:
Restart

Bertogim committed:
Now restore the viewer to the pull requested one
